website/docs/community/bindings.md

```diff
@@ -17,7 +17,9 @@ As a C library, bindings can be made to call H3 functions from different program
 
 ## BigQuery
 
-- [cartodb/bigquery-jslibs](https://github.com/CartoDB/bigquery-jslibs)
+- [CARTO Analytics Toolbox](https://docs.carto.com/analytics-toolbox-bigquery/sql-reference/h3/)
```

Comment:
I'd prefer to leave the docs out - many of the other linked projects have docs, but we link to the main page instead. I can see that you're trying to differentiate the DB-specific pieces, but this seems pretty clear from the main page - if we wanted this to be DB-specific, maybe link to https://github.com/CartoDB/analytics-toolbox-core#bigquery ?

Reply:
Well, the issue with Bigquery and snowflake is that you can have access to the functions without the need to install them, and that is probably what users want more. 

I have created a new section on the github repo on Getting Started so that the user knows they can start using them without doing anything, or they can install them directly on the DB.

https://github.com/CartoDB/analytics-toolbox-core/pull/250

My proposal is to link to their section, what do you think?

Reply:
Let me know what you think of this